packages/amazonq/test/e2e_new/amazonq/helpers/mcpHelper.ts

```diff
@@ -0,0 +1,308 @@
+        const buttonsWrapper = await navWrapper.findElement(By.css('.mynah-nav-tabs-bar-buttons-wrapper'))
+        const buttons = await buttonsWrapper.findElements(
+            By.css('.mynah-button.mynah-button-secondary.fill-state-always.mynah-ui-clickable-item')
+        )
+        for (const button of buttons) {
+            const icon = await button.findElement(By.css('i.mynah-ui-icon.mynah-ui-icon-tools'))
+            if (icon) {
+                await button.click()
+                await webviewView.getDriver().actions().move({ x: 0, y: 0 }).perform()
+                return true
+            }
+        }
+        console.log('Tools button not found')
+        return false
+    } catch (e) {
+        console.error('Error clicking tools button:', e)
+        return false
+    }
+}
+
+/**
+ * Clicks the add button in the MCP server configuration panel
+ * @param webviewView The WebviewView instance
+ * @returns Promise<boolean> True if add button was found and clicked, false otherwise
+ */
+export async function clickMCPAddButton(webviewView: WebviewView): Promise<boolean> {
+    try {
+        const sheetWrapper = await waitForElement(webviewView, By.id('mynah-sheet-wrapper'))
+        const header = await sheetWrapper.findElement(By.css('.mynah-sheet-header'))
+        const actionsContainer = await header.findElement(By.css('.mynah-sheet-header-actions-container'))
+        const addButton = await actionsContainer.findElement(By.css('button:has(i.mynah-ui-icon-plus)'))
+        await addButton.click()
+        return true
+    } catch (e) {
+        console.error('Error clicking the MCP add button:', e)
+        return false
+    }
+}
+
+/**
+ * Configures an MCP server with the provided settings
+ * @param webviewView The WebviewView instance
+ * @param config Configuration object with optional parameters
+ * @returns Promise<boolean> True if configuration was successful, false otherwise
+ * Note: I have the default settings in the defaultConfig
+ */
+interface MCPServerConfig {
+    scope?: 'global' | 'workspace'
+    name?: string
+    transport?: number
+    command?: string
+    args?: string[]
+    nameEnvironmentVariable?: string
+    valueEnvironmentVariable?: string
+    timeout?: number
+}
+
+const defaultConfig: MCPServerConfig = {
+    scope: 'global',
+    name: 'aws-documentation',
+    transport: 0,
+    command: 'uvx',
+    args: ['awslabs.aws-documentation-mcp-server@latest'],
+    nameEnvironmentVariable: 'hi',
+    valueEnvironmentVariable: 'hi',
+    timeout: 0,
+}
+
+const formItemsMap = {
+    SCOPE: 0,
+    NAME: 1,
+    TRANSPORT: 2,
+    COMMAND: 3,
+    ARGS: 4,
+    ENV_VARS: 6,
+    TIMEOUT: 9,
+} as const
+
+type McpFormItem = keyof typeof formItemsMap
+
+async function selectScope(container: WebElement, scope: string) {
+    try {
+        const a = await container.findElements(By.css('.mynah-form-input-radio-label.mynah-ui-clickable-item'))
+        if (scope === 'global') {
```

Comment:
What is this logic for? Also, are there more descriptive names than `a` and `b`?